sdks/python/src/opik/evaluation/tasks_scorer.py

```diff
@@ -59,11 +59,13 @@ def _process_item(
     item: dataset_item.DatasetItem,
     task: LLMTask,
     scoring_metrics: List[base_metric.BaseMetric],
+    project_name: str,
```

Comment:
after updating the code in the `evaluator.py`, it will become `project_name: Optional[str]`